sources.yaml

```diff
@@ -0,0 +1,87 @@
+# apiVersion: sources.knative.dev/v1alpha1
```

Comment:
What's this file for?

Reply:
I was using it to test the sources, and I thought it was useful since I don't know all their specs off the top of my head :)